tests/test_database.py

```diff
@@ -740,44 +775,29 @@ def test_update(tmpdir):
     other2.save(other2_root)
     audformat.testing.create_audio_files(other2, file_duration='0.1s')
 
-    # raises error because schemes do not match
-
-    with pytest.raises(ValueError):
-        audformat.utils.concat(
-            [db['table'].df, other1['table'].df],
-            overwrite=True,
-        )
-
-    # replace labels to avoid error
+    # update db with [other1, other2]
 
-    db.schemes['labels'].replace_labels(other1.schemes['labels'].labels)
```

Comment:
It's no longer required that we call `replace_labels()` here, since this is now handled by `Database.update()`.